package.json

```diff
@@ -45,6 +45,8 @@
   ],
   "homepage": "https://github.com/brave/web-discovery-project#readme",
   "resolutions": {
+    "ansi-regex": "5.0.1",
+    "ansi-html": "mahdyar/ansi-html-community#v0.0.8",
```

Comment:
ansi-html is no longer maintained so this repo does a one-line fix for it https://github.com/Tjatse/ansi-html/compare/master...mahdyar:master